### PR TITLE
suggest some schedule adjustments

### PR DESCRIPTION
**Pull request by q9f**
please don't merge until we discussed this

### CODE REVIEW

File: src/pages/program.jsx [hunk 1]
```diff
@@ -451,6 +451,16 @@ const Program = () => {
               title="Doors open"
               eventLocations={[locations.mainEnterence]}
             />
+            {isExtravaganza && (
+              <ProgramItem
+                dayStr="2022-09-16"
+                startTime="16:00"
```
Comment (q9f) [inline src/pages/program.jsx:457]: If I'm not mistaken, this will be up as soon as the doors open. Is it?

File: src/pages/program.jsx [hunk 2]
```diff
@@ -575,7 +575,7 @@ const Program = () => {
               <ProgramItem
                 dayStr="2022-09-16"
                 startTime="20:00"
-                endTime="22:00"
+                endTime="23:00"
```
Comment (q9f) [inline src/pages/program.jsx:578]: Shall we have the outdoor set till 11pm and then move inside?

File: src/pages/program.jsx [hunk 3]
```diff
@@ -657,7 +657,7 @@ Technical requirements: Users should feel comfortable using a terminal & Docker
             {isExtravaganza && (
               <ProgramItem
                 dayStr="2022-09-16"
-                startTime="23:59"
+                startTime="23:00"
```
Comment (q9f) [inline src/pages/program.jsx:660]: Michael could start at 11pm just like Saturday as Creator's Lab is not used after 11.

File: src/pages/program.jsx [hunk 4]
```diff
@@ -927,7 +927,7 @@ Technical requirements: Users should feel comfortable using a terminal & Docker
               <ProgramItem
                 dayStr="2022-09-18"
                 startTime="09:00"
-                endTime="16:00"
+                endTime="15:00"
```
Comment (q9f) [inline src/pages/program.jsx:930]: I think upper floors close at 3pm, do they?

File: src/pages/program.jsx (hunk 1 above)
Comment (kuzdogan) [reply src/pages/program.jsx:457]: I assumed a time myself, but actually still waiting for Griffin's response.

File: src/pages/program.jsx (hunk 2 above)
Comment (kuzdogan) [reply src/pages/program.jsx:578]: Wasn't it that we are allowed until 22?

Comment (q9f) [reply src/pages/program.jsx:578]: Yes but there is a grace period ;)

**Pull request merged**